### DISCUSSION TITLE
Let artists enable purchase of full discography

### DISCUSSION THREAD

**Discussion by simonv3** [funmusicplace/mirlo#587] [Ideas]
Similar to [Bandcamp's feature](https://get.bandcamp.help/hc/en-us/articles/360013673993-How-do-I-enable-the-purchase-of-my-full-digital-discography)

Some open question: 

* is this total amount paid set by the individual album prices (what if all albums are priced as pay what you want) 
* is there some threshold of "supporter" tier that gives people full access to the discography
* Is there an amount spent on an artist that automatically "unlocks" all future purchases

Considering some of those questions I do feel like the simplest is to just set a boolean + price on the artist account that lets the artist decide "if someone pays me this much money, they can download everything I have"

**Comment by Cwpute** [comment]
When buying all the releases in bulk, and if a specific price hasn't been set by the artist, please show the math behind the price.
If there are pay-what-you-want albums in the mix, i'd say add a specific line about it that would say somehting like: 
> Some releases in this bundle are pay-what-you-want. Would you like to pay a little higher to support these releases ?

...so that users clearly understand "pay-what-you-want" doesn't mean "free".

**Comment by simonv3** [comment]
This is implemented, and artists set a specific price. If they don't set a specific price it doesn't work. If changes are needed these can be part of a different ticket
